config/routes.rb

```diff
@@ -189,9 +189,9 @@
   post 'competitions/:id/disconnect_stripe' => 'competitions#disconnect_stripe', as: :competition_disconnect_stripe
 
   get 'panel' => 'panel#index'
-  get 'panel/delegate-crash-course', to: redirect('https://documents.worldcubeassociation.org/edudoc/delegate-crash-course/delegate_crash_course.pdf', status: 302)
```

Comment:
A lot of Delegates / ROs may have this link bookmarked or otherwise use this link in their internal documentation. I don't think we should remove it prematurely.

Reply:
Okay, so is there any risk associated with merging this PR right now?

Reply:
Nope, it's risk free.

Reply:
So what will happen with the countless ROs who may have bookmarked this link that you're deleting in this PR?

Reply:
Just chiming in on behalf of WQAC, I am comfortable making any necessary changes to #9308 that come as a result of this PR. Keeping the redirect unchanged here and having the discussion when #9308 is ready to merge would also keep this commit focused on the React changes.

Reply:
Thanks Nick, @gregorbg I've added back redirect at the end of routes.rb under "Deprecated Links" section.